spec/lib/reports/customers_report_spec.rb

```diff
@@ -274,18 +230,14 @@ module Customers
               expect(subject.filter(orders)).to eq(orders)
             end
 
-            it "returns orders with a specific supplier" do
-              supplier = create(:supplier_enterprise)
-              supplier2 = create(:supplier_enterprise)
-              product1 = create(:simple_product, supplier: supplier)
-              product2 = create(:simple_product, supplier: supplier2)
-              order1 = create(:order)
-              order2 = create(:order)
-              order1.line_items << create(:line_item, product: product1)
-              order2.line_items << create(:line_item, product: product2)
-
-              allow(subject).to receive(:params).and_return(supplier_id: supplier.id)
-              expect(subject.filter(orders)).to eq([order1])
+            it "filters to a specific completed_at date" do
+              o1 = create(:order, completed_at: 1.day.ago)
+              o2 = create(:order, completed_at: 3.days.ago)
+              o3 = create(:order, completed_at: 5.days.ago)
+
+              allow(subject).to receive(:params).and_return({ q: { completed_at_gt: 1.day.before(o2.completed_at),
+                                                                   completed_at_lt: 1.day.after(o2.completed_at) } })
```

Comment:
This may just be my personal preference but I like to keep the code further to the left:
```suggestion
              allow(subject).to receive(:params).and_return(
                q: {
                   completed_at_gt: 1.day.before(o2.completed_at),
                   completed_at_lt: 1.day.after(o2.completed_at)
                 }
               )
```

It's easier to add to and if something changes, like `subject` is renamed to `customers_report` then the indentation of the following lines don't have to be changed.

Reply:
Thanks! Yes, this is more readable !